src/extension.ts

```diff
@@ -175,7 +175,16 @@ export async function activate(context: vscode.ExtensionContext) {
 
 		vscode.commands.registerCommand('camelk.integrations.createNewIntegrationFile', async (...args:any[]) => { await NewIntegrationFileCommand.create(args);});
 		vscode.commands.registerCommand('camelk.integrations.selectFirstNode', () => { selectFirstItemInTree();});
-		vscode.commands.registerCommand('camelk.classpath.refresh', async (uri:vscode.Uri) => { await downloadSpecificCamelKJavaDependencies(context, uri, mainOutputChannel)});
+		vscode.commands.registerCommand('camelk.classpath.refresh', async (uri:vscode.Uri) => {
+			vscode.window.withProgress({
+				location: vscode.ProgressLocation.Notification,
+				title: "Updating the classpath...",
```

Comment:
would be nice to provide a more precise message something like:
"Updating the classpath of the Camel K Integration file..."

or maybe as a first iteration, as only the download of the libraries is awaited, something like: "Setting up dependencies for the Camel K integration file..."